doc/user/ANIMATIONS.md

```diff
@@ -34,10 +34,17 @@ F3D animation behavior can be fully controlled from the command line using the f
 | \-\-animation\-index=-1      |                     | Play all animations at once (.gltf/.glb only). |
 | \-\-animation\-speed\-factor | Time Unit = Seconds | Adjust time unit.                              |
 | \-\-animation\-frame\-rate   | 60 FPS              | Adjust animation frame rate.                   |
+| \-\-animation\-time          |                     | Load a specific time value on start.           |
 
 ## Animation Interactions
 - Press <kbd>W</kbd> to cycle through animations
 - Press <kbd>Space</kbd> to play/pause animation
 
 ## Time Units
-- When F3D plays an animation, it assumes the time unit is in seconds to show accurate speed of animation.
+
+When F3D plays an animation, it assumes the time unit is in seconds to show accurate speed of animation.
+
+## Coloring range
+
+When coloring while loading a time value or playing the animation, the coloring range will be automatically expanded from previously loaded time value,
```

Comment:
Shouldn't we just compute the range of all timesteps when the coloring is requested?

Reply:
This is not the right approach imo, as it will be 0(nTimeSteps). eg the following call would take a long time:

`f3d file_with_10000_ts.ex2 --animation-time=2 -sb`